index.bs

```diff
@@ -2979,11 +2979,22 @@ As such, monkey patching (wrongly!) presupposes that underlying functionality ca
 Wikipedia also describes some <a href="https://en.wikipedia.org/wiki/Monkey_patch#Pitfalls">additional pitfalls of monkey patching</a>.
 
 <h4 id="cant-avoid-monkey-patching">If you need to monkey patch</h4>
-Sometimes monkey patching is unavoidable (e.g., early in the design phase of a new specification). If you find yourself needing to monkey patch an existing specification, make sure you:
-
- 1. File a bug, or create a fix, for the existing specification. This will help you understand what the problem is and how to fix it, and maybe even alleviate you from even creating the monkey patch in the first place.
- 1. In your spec, clearly mark the monkey patch as a temporary workaround (e.g., using a big red box) and make it clear to readers that you intend to add your patch to the existing specification. Make sure you link to the existing specification and the bug or fix that you filed.
- 1. Work together with the authors of the existing specification on a solution. If the existing specification is no longer maintained... Congratulations! you are now the editor of that existing specification. If necessary, add suitable extension points to the existing specification to better handle your changes.
+Sometimes monkey patching is unavoidable (e.g., while a new specification is incubating in a community group). If you find yourself needing to monkey patch an existing specification, make sure you:
```

Comment:
It's not just that. E.g. in the CSS WG we often write specs this way in the very early stages where e.g. Level 3 is still being iterated on, but we're also working on a Level 4 draft.